internal/app/api/v1/testsuites.go

```diff
@@ -253,12 +253,19 @@ func (s TestkubeAPI) ListTestSuiteWithExecutionsHandler() fiber.Handler {
 		}
 
 		status := c.Query("status")
+		statusMap := make(map[testkube.TestSuiteExecutionStatus]struct{})
```

Comment:
Should we move it to some external package?

Reply:
yes, moved it to model package